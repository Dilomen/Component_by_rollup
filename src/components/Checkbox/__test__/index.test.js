import React from "react";
import { Checkbox } from "../index.js";
import { renderSnapshot, mountComponent } from "@tests";

mountComponent(<Checkbox></Checkbox>);

renderSnapshot(<Checkbox></Checkbox>);