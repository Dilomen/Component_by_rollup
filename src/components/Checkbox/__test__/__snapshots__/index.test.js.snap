// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component could be rendered snapshots without conflict render Snapshots 1`] = `
<label
  className="serein-checkbox-wrap "
>
  <input
    checked={false}
    disabled={false}
    onChange={[Function]}
    type="checkbox"
  />
  <span />
</label>
`;
